standardization/src/main/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParser.scala

```diff
@@ -159,13 +159,13 @@ object TypeParser {
     }
   }
 
-  private trait AtomicParser extends TypeParser {
+  private trait PrimitiveParser extends TypeParser {
     override def standardize(): ParseOutput = {
-      val castedCol: Column = atomicCastLogic
+      val castedCol: Column = assemblePrimitiveCastLogic
       val err = when((column isNull) and lit(!field.nullable),
         array(callUDF("stdNullErr", lit(s"$currentColumnPath${if (isArrayElement) "[*]" else ""}")))
       ).otherwise(
-        when((column isNotNull) and atomicCastErrorLogic(castedCol),
+        when((column isNotNull) and primitiveCastErrorLogic(castedCol),
```

Comment:
Also can be `assemblePrimitiveCastErrorLogic()`...

Reply:
OK, will use the name in next PR which changes exactly this code.